GothicVRProject/GothicVR
# Feature/267 dialog system

# To test
1. Open Lab.unity scene
2. Enable/Disable XRDeviceSimulator to your needs
3. Start game and walk into Bloodwyn (just run into him and it will trigger his dialog)
4. He should now talk to you
5. Check for
    6. Body is randomly animated
    7. Mouth is animated
    8. You are able to pick dialog options
9. If you can successfully close the dialog with "END" at the end, the test is successful.

Boundary (no feature for this branch):
* Dialog subtitles
* Dialogmenu is a black box somewhere close to Hero only

![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/5c8bb6aa-39fc-4e6a-bd4d-01254e7fe90d)
